include/mega/filesystem.h

```diff
@@ -22,7 +22,11 @@
 #ifndef MEGA_FILESYSTEM_H
 #define MEGA_FILESYSTEM_H 1
 
-#if defined (__linux__) || defined (__ANDROID__)
+#if defined (__linux__) && !defined (__ANDROID__)
+#include </usr/include/linux/magic.h>
```

Comment:
I think the `/usr/include` is redundant here, and it could be counterproductive, since the default include path does not necessarily need to be at `/usr/include`
```suggestion
#include <linux/magic.h>
```